xbmc/utils/Mp4ChplReader.h

```diff
@@ -0,0 +1,116 @@
+/*
+ *  Copyright (C) 2005-2025 Team Kodi
```

Comment:
(nit) For new files can you do

```
Copyright (C) 2025 Team Kodi
```

This gives the reader context for when the file was created. We don't want ppl thinking this is 20 year old code.